### PR TITLE
store/tikv: update tests with mocktikv

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by disksing** [comment]
@morefreeze @ngaut @shenli PTAL

File: store/tikv/mock-tikv/cluster.go [hunk 1]
```diff
@@ -187,6 +187,15 @@ func (c *Cluster) Split(regionID, newRegionID uint64, key []byte, leaderStoreID
 	c.regions[newRegionID] = newRegion
 }
 
+// Merge merges 2 Regions, their key ranges should be adjoining.
```
Comment (shenli) [inline store/tikv/mock-tikv/cluster.go]: adjoining -> adjacent

**Comment by shenli** [comment]
Rest LGTM

**Comment by morefreeze** [comment]
LGTM

**Pull request merged**